src/main/haskell/kore/src/Kore/Step/Simplification/Ceil.hs

```diff
@@ -118,39 +115,34 @@ makeEvaluate tools child
 makeEvaluateNonBoolCeil
     ::  ( MetaOrObject level
         , SortedVariable variable
-        , Show (variable level)
         , Ord (variable level)
+        , Show (variable level)
```

Comment:
Could we group these two constraints into a single one, DebugVariable or something similar (not necessarily in this PR)? I guess that's their purpose, so we should be explicit about what happens.

Reply:
Yes, I will make a separate pull request for that. I think it should be a class for any syntactic object, not just variables; the `Unparse (variable level)` is just getting pulled in to satisfy an `Unparse (Pattern level domain variable child)` instance somewhere. But at this location, the constraint could just be `DebugSyntax (variable level)`.